packages/flutter/lib/src/painting/text_painter.dart

```diff
@@ -351,6 +356,22 @@ class TextPainter {
     _needsLayout = false;
   }
 
+  void layoutToMaxIntrinsicWidth({ double maxWidth }) {
```

Comment:
if you made maxWidth default to infinity, you wouldn't need to have an ugly infinity at the call sites. :-)